app/components/chatBot/ChatBot.tsx

```diff
@@ -109,29 +119,76 @@ const ChatBot =()=> {
         },
       ]);
     } catch (error) {
-      console.error('Error fetching response from ChatGPT:', error);
+      console.error('Error fetching response from ChatGPT:', error)
     } finally {
       setIsTyping(false);
     }
   }
 
+  /* useEffect(() => {
+    // Hide the pop-up when the chatbot is opened
+    if (isOpen) {
+      setShowPopup(false);
+    } else {
+      // If not opened, show pop-up for 5 seconds
+      const timer = setTimeout(() => setShowPopup(false), 20000);
+      return () => clearTimeout(timer);
+    }
```

Comment:
hello @ouassimaelyakoubi  good job so far , I would like the code base to be cleaner , it is always good to remove unused /commented code , overall  keep up the good work!